src/main/webapp/code-first/openapi.json

```diff
@@ -1,380 +1,492 @@
+      "put": {
+        "tags": [
+          "pet"
+        ],
+        "summary": "Update an existing pet.",
+        "description": "Update an existing pet by Id",
+        "operationId": "updatePet",
+        "requestBody": {
+          "description": "Pet object that needs to be updated in the store",
+          "content": {
+            "application/json": {
+              "schema": {
+                "$ref": "#/components/schemas/Pet",
+                "description": "A Pet in JSON Format",
+                "required": [
+                  "id"
+                ],
+                "writeOnly": true
               }
             },
-            "application/xml" : {
-              "schema" : {
-                "$ref" : "#/components/schemas/Pet",
-                "description" : "A Pet in XML Format",
-                "required" : [ "id" ],
-                "writeOnly" : true
+            "application/xml": {
+              "schema": {
+                "$ref": "#/components/schemas/Pet",
+                "description": "A Pet in XML Format",
+                "required": [
+                  "id"
+                ],
+                "writeOnly": true
               }
             }
           },
-          "required" : true
+          "required": true
         },
-        "responses" : {
-          "200" : {
-            "description" : "Successful operation",
-            "content" : {
-              "application/xml" : {
-                "schema" : {
-                  "$ref" : "#/components/schemas/Pet",
-                  "description" : "A Pet in XML Format",
-                  "readOnly" : true
+        "responses": {
+          "200": {
+            "description": "Successful operation",
+            "content": {
+              "application/xml": {
+                "schema": {
+                  "$ref": "#/components/schemas/Pet",
+                  "description": "A Pet in XML Format",
+                  "readOnly": true
                 }
               },
-              "application/json" : {
-                "schema" : {
-                  "$ref" : "#/components/schemas/Pet",
-                  "description" : "A Pet in JSON Format",
-                  "readOnly" : true
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Pet",
+                  "description": "A Pet in JSON Format",
+                  "readOnly": true
                 }
               }
             }
           },
-          "400" : {
-            "description" : "Invalid ID supplied"
+          "400": {
+            "description": "Invalid ID supplied"
           },
-          "404" : {
-            "description" : "Pet not found"
+          "404": {
+            "description": "Pet not found"
           },
-          "405" : {
-            "description" : "Validation exception"
+          "405": {
+            "description": "Validation exception"
+          },
+          "default": {
```

Comment:
I've noticed there is a pattern of copying `200` response to `default` response (exactly copy) - is there a reasoning/rule behind this?

Reply:
So maybe defining a success response in `components/responses` and referencing both in `200` and `default`? I mean just to avoid duplicating it. This might make reading the API Description a bit harder, but there are always pros and cons...

Reply:
I'll come up with a less verbose way of satisfying the rules without duplication